tap_shopify/streams/graphql/gql_base.py

```diff
@@ -53,7 +53,8 @@ def get_query(self):
         """
         raise NotImplementedError("Function Not Implemented")
 
-    def transform_object(self, obj):
+    @staticmethod
+    def transform_object(obj):
```

Comment:
why do we need this method to be static?

this method is modified in the stream metafields, where the implementation uses a object reference of the class.